tools/ktx/command_create.cpp

```diff
@@ -611,10 +621,14 @@ Create a KTX2 file from various input files.
     The following options are available:
     <dl>
         <dt>--format &lt;enum&gt;</dt>
-        <dd>KTX format enum. The enum names are matching the VkFormats without the VK_FORMAT_ prefix.
+        <dd>KTX format enum that specifies the image data format.
+            The enum names are matching the VkFormats without the VK_FORMAT_ prefix.
+            The VK_FORMAT_ prefix is ignored if present.
+            If the --encode option is used it specifies the format created from the
+            the input files before the encoding step.
```

Comment:
Please change "If the --encode option is used" to what you wrote in your response to issue #726. 

```
When used with --encode it specifies the format of your input files before the encoding step. In this case
it must be one of:
R8_UNORM
R8_SRGB
R8G8_UNORM
R8G8_SRGB
R8G8B8_UNORM
R8G8B8_SRGB
R8G8B8A8_UNORM
R8G8B8A8_SRGB
```

Then, after the sentence about ASTC,

```
The format will be used to verify and load all your input files into a texture before encoding.
```